app/controllers/general.php

```diff
@@ -250,7 +250,8 @@ function router(App $utopia, Database $dbForConsole, callable $getProjectDB, Swo
             'APPWRITE_FUNCTION_PROJECT_ID' => $project->getId(),
             'APPWRITE_FUNCTION_RUNTIME_NAME' => $runtime['name'] ?? '',
             'APPWRITE_FUNCTION_RUNTIME_VERSION' => $runtime['version'] ?? '',
-            'APPWRITE_VERSION' => APP_VERSION_STABLE
+            'APPWRITE_VERSION' => APP_VERSION_STABLE,
+            'APPWRITE_ENV' => 'production',
```

Comment:
why is this hard coded? There is an environment variable for this. 
https://github.com/appwrite/appwrite/blob/main/app/config/variables.php#L11
This value should be used

Reply:
I know it was hard-coded for CLI. @Meldiron can we use the variable?